src/applications/lgy/coe/containers/ConfirmationPage.jsx

```diff
@@ -12,41 +15,76 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const { submission, data } = this.props.form;
-    const { response } = submission;
-    const name = data.veteranFullName;
+    // const { submission, data } = this.props.form;
+    // const { response } = submission;
+    // const name = data.veteranFullName;
+
+    const name = {
```

Comment:
Are we meaning to not use the form data and instead make this static?

Reply:
Fixed